docs/runtime/compiler-ir.md

```diff
@@ -12,12 +12,18 @@ for future goals.
 ## Visualization
 
 The IR can be visualized using `--vm.D=enso.compiler.dumpIr` system property.
```

Comment:
The advantage of the `.dot` format is support for multiple tools that can read it. That's lost with the `.html` files. They serve only one purpose - to be viewed in a browser. Maybe:

- `--vm.D=enso.compiler.dumpIr=dot`
- `--vm.D=enso.compiler.dumpIr=html`